### PR TITLE
Backend: One useState for the form

### PR DESCRIPTION
**Pull request by GeorgeHarland**
Also fixed temp_all_contents function. Now properly returns all data from the table

Frontend: https://github.com/kristian-welsh/character-creation-frontend/compare/feat/data-in-one-object?expand=1

### CODE REVIEW

File: docs/dbsetup.txt
Comment (kristian-welsh) [inline docs/dbsetup.txt:3]: good doc canges

File: server.py
Comment (kristian-welsh) [inline server.py:24]: This makes it work wheen running locally at. the expence of docker changes needed in the future. it's a trade off, probably a good one for right now.

File: server.py
Comment (kristian-welsh) [inline server.py:81]: I'm surprised that both of these work, you tested the new one?

**Review by kristian-welsh — COMMENTED** [review]
mostly looks good, small question about the change to collection.find()

File: server.py
Comment (GeorgeHarland) [reply server.py:81]: the deleted one didn't work (maybe after these changes). latter works, frontend logs the object of all the fields in the table

Comment (kristian-welsh) [reply server.py:81]: nice nice. I'll approve, wait for both PRs approvals before merging at the same time

**Review by kristian-welsh — APPROVED** [review]

**Pull request merged**
